src/components/CurrentSong.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import PropTypes from "prop-types";
+import DefaultCover from "../assets/cover_placeholder.gif";
```

Comment:
I think for now we can use the CDN for the image.

I would like to change this image to an SVG so we can save another request.

Reply:
Hmm, not really sure I understand your comment. 

I downloaded the gif and stored it in the `/assets`. Hence the import is `from "../assets/cover_placeholder.gif"`. You can click the `Files changed` tab in the pull request, then see the first change.

Reply:
I'm planning on changing this image to SVG element. Will do in a separate PR.

Reply:
Thank you for your patience @huyenltnguyen, @xyozio.
Is there a specific reason why you prefer having  the cover_placeholder.gif locally rather than on the CDN?

Reply:
@ahmadabdolsaheb I don't really have an opinion on local storing vs CDN. I just thought that while we serve the SVGs in `/assets`, we could have the gif in there as well, since it's a media file also.

But I don't have experience in the cost / performance area, so I'll let you guys decide on this one.

Reply:
@huyenltnguyen, thank you for the quick response.
It would be great if you could leave out this modification for this pr.
We could discuss a general strategy for assets in a dedicated issue/pr.

Reply:
Done 😄